app/src/main/java/org/wikipedia/userprofile/UserContributionsStats.kt

```diff
@@ -53,73 +56,52 @@ object UserContributionsStats {
         val qLangMap = HashMap<String, HashSet<String>>()
 
         for (userContribution in response.query()!!.userContributions()) {
-            var descLang = ""
-            val strArr = userContribution.comment.split(" ")
-            for (str in strArr) {
-                if (str.contains("wbsetdescription")) {
-                    val descArr = str.split("|")
-                    if (descArr.size > 1) {
-                        descLang = descArr[1]
-                        break
-                    }
-                }
-            }
-            if (descLang.isEmpty()) {
+            val descLang = userContribution.comment.split(" ")
+                    .filter { "wbsetdescription" in it }
+                    .flatMap { it.split("|") }
+                    .firstOrNull()
```

Comment:
We should get the second item from the splitted array. Maybe use `getOrNull(1)`?

Reply:
I didn't know that, thanks!